app/src/ui/lib/button.tsx

```diff
@@ -199,9 +202,9 @@ export class Button extends React.Component<IButtonProps, {}> {
         {tooltip && (
           <Tooltip
             target={this.innerButtonRef}
-            direction={TooltipDirection.NORTH}
+            direction={this.props.toolTipDirection ?? TooltipDirection.NORTH}
             // Show the tooltip immediately on hover if the button is disabled
-            delay={disabled && tooltip ? 0 : undefined}
+            delay={disabled && Tooltip ? 0 : undefined}
```

Comment:
I think this `Tooltip` (or even the original `tooltip`) is unnecessary because it will always be `true` here, right?
```suggestion
            delay={disabled ? 0 : undefined}
```

Reply:
Hmm.. must of been when I was toying with how to get the tooltip to open on focus... which it does now.